pkg/apis/rbac/validation/policy_comparator.go

```diff
@@ -69,9 +73,11 @@ func breakdownRule(rule rbac.PolicyRule) []rbac.PolicyRule {
 		}
 	}
 
-	// Non-resource URLs are unique because they don't combine with other policy rule fields.
+	// Non-resource URLs are unique because they only combine with verbs.
 	for _, nonResourceURL := range rule.NonResourceURLs {
```

Comment:
Don't non-resource URLS have a different set of verbs than resource URLs.  Does that get validated somewhere?  Or are verbs not validated at all, since they can be dynamic?